src/controllers/expenses.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const expensesServices = require('../services/expenses');
+const usersServices = require('../services/users');
+
+function getAll(req, res) {
+  const queryParams = req.query;
+
+  const expenses = expensesServices.getAll(queryParams);
+
+  res.statusCode = 200;
+  res.send(expenses);
+}
+
+function getById(req, res) {
+  const { expenseId } = req.params;
+
+  if (!expenseId) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundExpense = expensesServices.getById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundExpense);
+}
+
+function add(req, res) {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+  } = req.body;
+
+  const foundUser = usersServices.getById(userId);
+
+  const isAllDataProvided = userId
+    && spentAt
+    && title
+    && amount
+    && category;
+
+  const isAllDataValid = foundUser
```

Comment:
`isAllDataValid` made the same as `isAllDataProvided`, cause if some field will be missed, it will not match your check.
For example, if there will be no `title`, it would not match '`string`' type